ui/package.json

```diff
@@ -31,5 +31,15 @@
     "typescript-eslint": "^8.26.1",
     "vite": "^6.2.1",
     "vite-plugin-solid": "^2.11.6"
+  },
+  "pnpm": {
+    "ignoredBuiltDependencies": [
+      "cypress",
+      "esbuild"
+    ],
+    "onlyBuiltDependencies": [
+      "cypress",
+      "esbuild"
+    ]
   }
```

Comment:
waar is dit voor?

Reply:
Allows cypress to be build during installation https://pnpm.io/package_json#pnpmonlybuiltdependencies
Heeft ie automatisch toegevoegd tho dus kan ik er wel uit halen